xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.xml

```diff
@@ -80,6 +80,8 @@ rendering.macro.menu.name=Menu
 rendering.macro.menu.description=Displays a menu created using simple wiki syntax (nested lists and links).
 rendering.macro.menu.parameter.id.name=Id
 rendering.macro.menu.parameter.id.description=Optional menu identifier that will be set on the HTML element that wraps the menu. You can use this identifier in JavaScript code to access the menu DOM or in functional tests to assert the menu items.
+rendering.macro.menu.parameter.label.name=Label
+rendering.macro.menu.parameter.label.description=Optional menu label used to describe the content of the menu.
```

Comment:
I would add more context, in particular that this label is important for accessibility as otherwise I think admins won't understand why they should set this label.